ibis/pandas/execution/tests/test_timecontext.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+import pandas.util.testing as tm
+import pytest
+
+import ibis
+import ibis.common.exceptions as com
+
+pytestmark = pytest.mark.pandas
+
+
+def test_execute_with_timecontext(time_table):
+    expr = time_table
+    # define a time context for time-series data
+    context = ('20170101', '20170103')
+
+    # without time context, execute produces every row
+    df_all = expr.execute()
+    assert len(df_all['time']) == 8
+
+    # with context set, execute produces only rows within context
+    df_within_context = expr.execute(timecontext=context)
+    assert len(df_within_context['time']) == 1
+
+
+merge_asof_minversion = pytest.mark.skipif(
```

Comment:
you can remove this we require a more recent version of pandas

Reply:
Glad to know, just removed it.